pyproject.toml

```diff
@@ -59,5 +59,5 @@ pythonPlatform = "Linux"
 reportUnknownParameterType = true
 
 [tool.codespell]
-skip = ".git,.tox,build,venv*"
+skip = ".git,.tox,build,venv*,tls_certificates.py"
```

Comment:
Does codespell have an analogue to a `.wokeignore` file?

Reply:
Nope afaik; configs need to live in `pyproject.toml` or in the command invocation itself